docs/getting-started/components/authz_manager.md

```diff
@@ -1,5 +1,5 @@
 # Authorization Manager
-An Authorization Manager is an instance of the `AuthManager` class that is plugged into one of the Feast servers to extract user details from the current request and inject them into the [permissions](../../getting-started/concepts/permissions.md) framework.
+An Authorization Manager is an instance of the `AuthManager` class that is plugged into one of the Feast servers to extract user details from the current request and inject them into the permission framework.
```

Comment:
why remove link?

Reply:
original link is broken (extra s in the file permissions.md). Thought we moved that doc is moved but with your comment realized it is type or some refactoring. now fixed it with the correct link.